app/src/main/java/com/example/tung/moviedb_07/screen/movie_detail/MovieDetailViewModel.java

```diff
@@ -1,9 +1,18 @@
 package com.example.tung.moviedb_07.screen.movie_detail;
 
+import android.content.Context;
 import android.view.View;
 import com.example.tung.moviedb_07.data.model.Genre;
 import com.example.tung.moviedb_07.data.model.Movie;
+import com.example.tung.moviedb_07.data.model.ProductionCompany;
+import com.example.tung.moviedb_07.data.source.MovieRepository;
+import com.example.tung.moviedb_07.data.source.local.sqlite.MovieLocalDataSource;
```

Comment:
ERROR: Unused import: com.example.tung.moviedb_07.data.source.local.sqlite.MovieLocalDataSource. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports